create-cluster.rb

```diff
@@ -75,6 +76,15 @@ def install_components(cluster_name)
   end
 end
 
+def disable_alerts
+  # This will replace the value of pagerduty_config to a dummy value in your working terraform.tfvars file.
```

Comment:
I'd suggest something like this;

> This will disable high-priority pagerduty alarms for your cluster, by replacing the XXXX with a dummy value

...and something similar for the slack webhook.

I don't think the "comment this to enable..." line is necessary, as long as it's clear what effect the code has.